inheritance/lib/src/main/java/inheritance/Shop.java

```diff
@@ -3,7 +3,7 @@
 import java.util.ArrayList;
 
 public class Shop {
-    priate String;
+    private toString();
     name;
     private String description;
     Private String;
```

Comment:
`private` must be lower case, and the variable name should be on the same line: `private String priceCategory`